DESCRIPTION

```diff
@@ -69,3 +71,4 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1.9001
 SystemRequirements: pandoc
+Remotes: dmurdoch/htmlwidgets@rglpatch, dmurdoch/downlit@rglpatch
```

Comment:
So we should be able to change this to `Remotes: dmurdoch/downlit` and everything will work?

Reply:
Yes, it should be fine now.

Reply:
To clarify:  by "everything will work" I mean nothing will break.   HTML widgets other than rgl's won't print until `htmlwidgets` is merged too, or the package providing them adapts to the new code.

Reply:
Yeah, that's what I meant, sorry.